gst/nnstreamer/nnstreamer_plugin_api_decoder.h

```diff
@@ -60,18 +60,24 @@ typedef struct _GstTensorDecoderDef
 
 /* extern functions for subplugin management, exist in tensor_decoder.c */
 /**
- * @brief decoder's subplugins should call this function to register
- * @param[in] decoder The decoder subplugin instance
+ * @brief Decoder's sub-plugins should call this function to register
+ * @param[in] decoder The decoder sub-plugin instance
  */
-extern gboolean
+extern int
 nnstreamer_decoder_probe (GstTensorDecoderDef * decoder);
 
 /**
- * @brief decoder's subplugin may call this to unregister
- * @param[in] name the name of decoder (modename)
+ * @brief Decoder's sub-plugin may call this to unregister
+ * @param[in] name The name of decoder sub-plugin
  */
 extern void
-nnstreamer_decoder_exit (const gchar * name);
+nnstreamer_decoder_exit (const char *name);
 
+/**
+ * @brief Find decoders sub-plugin with the name
+ * @param[in] name The name of decoder sub-plugin
+ */
+extern const GstTensorDecoderDef *
+nnstreamer_decoder_find (const char *name);
```

Comment:
Please add return description to the doxygen entry.

Reply:
updated dox entry